linkerd/core/src/main/scala/io/buoyant/linkerd/telemeter/UsageDataTelemeter.scala

```diff
@@ -117,13 +125,16 @@ private[telemeter] object UsageDataTelemeter {
     counters.toSeq
   }
 
+  type HttpRouter = { def identifier: Option[Seq[{ def kind: String }]] }
```

Comment:
i know that these can be costly at runtime. maybe worth a small note that this type is only used during configuration and not in the serving path?